pom.xml

```diff
@@ -121,6 +122,8 @@
     <suite.name>org.apache.carbondata.cluster.sdv.suite.SDVSuites</suite.name>
     <script.exetension>.sh</script.exetension>
     <carbon.hive.based.metastore>false</carbon.hive.based.metastore>
+    <grpc.version>1.10.0</grpc.version>
+    <netty.version>4.0.43.Final</netty.version>
```

Comment:
Better use netty version same version as spark uses. Otherwise lot of exclusions need to be done.

Reply:
yes. I checked spark2.2, they are using netty 4.0.43. see https://github.com/apache/spark/blob/branch-2.2/pom.xml#L556

because carbon-store-search module does not depend on spark, so I need to explicitly set it here

Reply:
I find that even the oldest version of grpc-netty depends on netty-codec-http2 which came out since 4.1.x. 

see  http://mvnrepository.com/artifact/io.grpc/grpc-netty